CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ IF (KokkosKernels_INSTALL_TESTING)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(test_common)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(perf_test)
   KOKKOSKERNELS_ADD_TEST_DIRECTORIES(unit_test)
+  KOKKOSKERNELS_ADD_EXAMPLE_DIRECTORIES(example)
```

Comment:
When doing install testing we force testing on. Does it make sense to do the same for examples?
````
SET(KOKKOSKERNELS_ENABLE_EXAMPLES ON)
````